lib/active_record/virtual_attributes/virtual_fields.rb

```diff
@@ -101,6 +101,35 @@ def merge_includes(hash1, hash2)
   end
 end
 
+def assert_klass_has_instance_method(klass, instance_method)
+  begin
+    klass.instance_method(instance_method)
+  rescue NameError => err
+    msg = "#{klass} is missing the method our prepended code is expecting to patch. Was the undefined method removed or renamed upstream?\nSee: #{__FILE__}.\nThe NameError was: #{err}. "
+    raise NameError, msg
```

Comment:
We will have to bump the minor version for this anyway with adding rails 7 support but since we're already dropping 6.0 and older, we can include this and just bump the major version.

Reply:
remember, this is not semver, we need to follow versions that signify this is good for rails 7.0.x - so we'll be using 7.0.y (y is not x, x is theirs, y is ours)